### DISCUSSION TITLE
deposit problem

### DISCUSSION THREAD

**Discussion by sasani65** [Chia-Network/chia-blockchain#15720] [Support]
hello i deposited my chia from exchange to your native wallet 8days ago and everything was correct but i cant see it in my asset.can you help me?
![photo_2023-07-09_16-56-52](https://github.com/Chia-Network/chia-blockchain/assets/139056160/33cfd94c-3597-4e02-ab5e-af24c11eea8c)

**Comment by BrandtH22** [comment]
Hey @sasani65, I am sorry for the delayed response. While moving our support from keybase to discord (https://discord.gg/chia) we have unfortunately not given github issues the attention they deserve and are working to be better at this.

Regarding the transaction it seems that the funds have successfully made it to the wallet ID you have listed in the screenshot. Can be verified using a blockchain explorer and entering that xch address, I had confirmed on alltheblocks (https://alltheblocks.net) and also on spacescan (https://www.spacescan.io/). Each is a third party explorer that has not be verified for security, accuracy, or performance by CNI but both are reporting that the 3.1515xch has been received in that xch address and has not yet been spent from it.

Have you seen the funds enter your wallet?
